lib/controller.dart

```diff
@@ -12,8 +12,29 @@ Map<String, model.Link> _navLinks = {
   'settings': model.Link('settings', 'Settings', view.renderSettingsTab)
 };
 
+Map<String, String> _errorMessages = {
```

Comment:
Consider using string constants instead of a map?

Reply:
Ah yes, Fixed